Don't discard CFLAGS of the environment and turn off all optimizations as
those are handled by our build system.

Author: Faidon Liambotis <paravoid@debian.org>

--- a/Makefile
+++ b/Makefile
@@ -83,7 +83,7 @@
 	rose_qsig_mwi.lo \
 	rose_qsig_name.lo \
 	version.lo
-CFLAGS=-Wall -Werror -Wstrict-prototypes -Wmissing-prototypes -g -fPIC $(ALERTING) $(LIBPRI_COUNTERS) $(LIBPRI_OPT)
+CFLAGS += -Wall -Werror -Wstrict-prototypes -Wmissing-prototypes -g -fPIC $(ALERTING) $(LIBPRI_COUNTERS)
 INSTALL_PREFIX=$(DESTDIR)
 INSTALL_BASE=/usr
 libdir?=$(INSTALL_BASE)/lib
@@ -109,16 +109,6 @@
 
 PRIVERSION:=$(shell GREP=$(GREP) AWK=$(AWK) build_tools/make_version .)
 
-#The problem with sparc is the best stuff is in newer versions of gcc (post 3.0) only.
-#This works for even old (2.96) versions of gcc and provides a small boost either way.
-#A ultrasparc cpu is really v9 but the stock debian stable 3.0 gcc doesnt support it.
-ifeq ($(PROC),sparc64)
-PROC=ultrasparc
-LIBPRI_OPT = -mtune=$(PROC) -O3 -pipe -fomit-frame-pointer -mcpu=v8
-else
-LIBPRI_OPT = -O2
-endif
-
 ifeq ($(CPUARCH),i686)
 CFLAGS += -m32
 SOFLAGS += -m32
